lib/adhearsion/asterisk/call_controller_methods.rb

```diff
@@ -403,7 +403,8 @@ def say_characters(argument)
       def play_tones(argument, wait = false)
         tones = [*argument].join(",")
         execute("Playtones", tones).tap do
-          sleep tones.scan(/(?<=\/)\d+/).map(&:to_i).sum.to_f / 1000 if wait
+          # sleep tones.scan(/(?<=\/)\d+/).map(&:to_i).reduce(0, :+).to_f / 1000 if wait # adhearsion:adhearsion-3
+          sleep tones.scan(/(?<=\/)\d+/).inject(0) { |sum, el| sum + el.to_i }.to_f / 1000 if wait # kares:ahn3
```

Comment:
the choice between these two feels like a toss up.  benchmarking shows that the kares version outperforms the adhearsion version during "warmup" more often than not, but overall times show both versions perform and win about the same number of executions.

e.g., 
```
Benchmark:
Rehearsal --------------------------------------------------
tones_scan_adhearsion_string   3.290000   0.150000   3.440000 (  2.536297)
tones_scan_kares_string        2.520000   0.020000   2.540000 (  2.208699)
tones_scan_adhearsion_array    2.480000   0.010000   2.490000 (  2.303453)
tones_scan_kares_array         2.420000   0.010000   2.430000 (  2.278088)
---------------------------------------- total: 10.900000sec

                     user     system      total        real
tones_scan_adhearsion_string   2.230000   0.000000   2.230000 (  2.210091)
tones_scan_kares_string        2.230000   0.010000   2.240000 (  2.201871)
tones_scan_adhearsion_array    2.340000   0.000000   2.340000 (  2.314568)
tones_scan_kares_array         2.370000   0.000000   2.370000 (  2.341170)
```

Reply:
Thanks for doing this, mate.

Sounds like it's a toss-up.  What's your pref?  I like the map-reduce approach, but I'm interested in your pref and think both are good.

Reply:
yea - reads simpler

Reply:
@gfaza Coolio, thanks man. 👍 

@kares Just FYI, in merging, we chose the adhearsion flavor of 

```ruby
          sleep tones.scan(/(?<=\/)\d+/).map(&:to_i).reduce(0, :+).to_f / 1000 if wait # adhearsion:adhearsion-3
```
instead of
```ruby
          sleep tones.scan(/(?<=\/)\d+/).inject(0) { |sum, el| sum + el.to_i }.to_f / 1000 if wait # kares:ahn3
```

Both seemed very performant.  Let us know if you have any concerns (even if we've already merged this).  Thanks man.